security-command-center/snippets/src/test/java/v2/FindingsIT.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2;
+
+import static com.google.common.truth.Truth.assertThat;
+import static com.google.common.truth.Truth.assertWithMessage;
+
+import com.google.cloud.securitycenter.v2.Finding;
+import com.google.cloud.securitycenter.v2.Source;
+import com.google.cloud.testing.junit4.MultipleAttemptsRule;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import v2.findings.GroupFindings;
+import v2.findings.GroupFindingsWithFilter;
+import v2.findings.ListAllFindings;
+import v2.findings.ListFindingsWithFilter;
+
+// Test v2 Findings samples.
+@RunWith(JUnit4.class)
+public class FindingsIT {
+
+  // TODO(Developer): Replace the below variables.
+  private static final String ORGANIZATION_ID = System.getenv("SCC_PROJECT_ORG_ID");
```

Comment:
nit: it is a bit unconventional to use external environment variable for integration testing. Where it is defined? How is it ensured that this will work on every environment? If we use the "fixed" test project, can we just setup the "fixed" org id?

Reply:
Thanks..

The org id is required here to account for the special testing permissions. These are fetched from Secret manager and are consistent in every run. More in the internal thread.